isa.c

```diff
@@ -11,6 +12,92 @@
 
 static struct CPUState next_state, curr_state;
 
+bool condition_check(uint8_t cond)
+{
+	if(cond == 0x00)	//EQ
```

Comment:
enums would be cool for 0x00 etc in conditions, though not compulsory
if you feel like doing it, go ahead, you'll get brownie points :)